test/unit/app.spec.js

```diff
@@ -37,6 +37,26 @@ describe('App', function() {
     });
   });
 
+  describe('when restarting the app', function() {
```

Comment:
I think it might be due to not being familiar with sinon/mocha, but my attempts at checking `stop` and `start are being thwarted in these tests. Continuing to look into it.